service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java

```diff
@@ -636,11 +636,8 @@ public void setTariffLimits(Long tariffId, SetTariffLimitsDto setTariffLimitsDto
                 throw new BadRequestException(ErrorMessage.MAX_BAG_VALUE_IS_INCORRECT);
             }
 
-            tariffsInfo.setMinPriceOfOrder(null);
-            tariffsInfo.setMaxPriceOfOrder(null);
-
-            tariffsInfo.setMinAmountOfBigBags(setTariffLimitsDto.getMinAmountOfBigBags());
-            tariffsInfo.setMaxAmountOfBigBags(setTariffLimitsDto.getMaxAmountOfBigBags());
+            tariffsInfo.setMinQuantity(setTariffLimitsDto.getMinAmountOfBigBags());
```

Comment:
Should we also rename `minAmountOfBigBags` to `minQuantity` in `SetTariffLimitsDto`?

Reply:
I'm not responsible about SetTariffLimitsDto, because I'm working on setLimitsByAmountOfBags, setLimitsBySumOfOrder
I have changes on that because the entity TariffsInfo has been changed